pages/Monster-Jerky.md

```diff
@@ -0,0 +1,3 @@
+*"No longer hungry"*
+
+Monster Jerky is a [Food](Food.md) item added by Slimefun. When consumed, it removes any Hunger status effect and adds Saturation I for 0.25 seconds.
```

Comment:
Also just a tip:
Try to work with line-breaks to make content go on the next line:
The syntax is:
`<br>`